src/main/java/com/gmail/service/ShcedulerService.java

```diff
@@ -0,0 +1,39 @@
+package com.gmail.service;
+
+import com.gmail.model.User;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDate;
+import java.util.List;
+
+/**
+ * Created by Space on 09.05.2019.
+ */
+@Service
+@Log4j
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class ShcedulerService {
+    private final UserService userService;
+    private final EmailService emailService;
+
+    @Scheduled(cron = "* * * * * *")
```

Comment:
Please move cron value to the property file.

Reply:
I move it to the property file. I don't know why it here

Reply:
Sorry

Reply:
So please fix it